lib/cli/src/commands/run.rs

```diff
@@ -54,6 +54,12 @@ pub struct Run {
     #[structopt(flatten)]
     wasi: Wasi,
 
+    /// Allow the Wasm module to be executed in an unsandboxed way with
+    /// the `Emscripten` ABI.
+    #[cfg(feature = "emscripten")]
+    #[structopt(long = "allow-emscripten")]
+    allow_emscripten: bool,
```

Comment:
I think a flag like `--emscripten` might be better and simpler to use for the user?

Reply:
I agree, the only reason I went with `--allow-emscripten` was that `--emscripten` sounds like it _has_ to be `emscripten` or it will fail, whereas `allow-emscripten` makes the distinction that nothing is being forced to be anything, just that something else is now allowed.

Reply:
I agree with @MarkMcCaskey here.
Also, I take this opportunity to remind that I'm a bit afraid of addings flags and options one by one until having a long list of non-sense options. Hence this proposal https://github.com/wasmerio/wasmer/issues/1237.

It's not a blocker for this PR though!